magefile.go

```diff
@@ -21,6 +21,7 @@ import (
 	. "get.porter.sh/porter/mage/docker"
 	"get.porter.sh/porter/mage/releases"
 	"get.porter.sh/porter/pkg/cnab"
+	"github.com/carolynvs/magex/ci"
```

Comment:
Just a general question what's the determining factor on if something goes here versus in the getporter/magefiles repo? Should we be moving some of this code into that repo?